### PR TITLE
Searches

### PR DESCRIPTION
**Pull request by adambraimbridge**

### CODE REVIEW

**Comment by adambraimbridge** [comment]
Hm, travis is reporting that it's failing on jQuery and lodash. I'll review tomorrow.

File: client/queries/searchterms.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+var daysFromNow = function (offset) {
+	offset = offset || 0;
+	var dateObject = new Date();
+	dateObject.setDate(dateObject.getDate() + offset);
+	return dateObject.toISOString();
+};
+
+var query = new Keen.Query("count", {
+	eventCollection: "dwell",
+	filters: [{"operator":"eq","property_name":"page.location.type","property_value":"search"},{"operator":"eq","property_name":"user.isStaff","property_value":false}],
```
Comment (commuterjoy) [inline client/queries/searchterms.js]: formatting everything on one line is a pain to read

File: client/queries/searchterms.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	groupBy: "page.location.search.q",
+	interval: "weekly",
+	targetProperty: "user.erights",
+	timeframe: "this_2_months",
```
Comment (commuterjoy) [inline client/queries/searchterms.js]: don't think you need `timeframe` here

**Comment by commuterjoy** [comment]
@adambraimbridge just needed 046b8e4

**Comment by commuterjoy** [comment]
I include jquery and lodash in the html page.

**Pull request merged**